style.css

```
@@ -0,0 +1,42 @@
+body {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    justify-content: center;
```

```suggestion
    display: flex;
    justify-content: center;
    font-family: Arial, Helvetica, sans-serif;
```

chyba chciałeś alfabetycznie, ale lepiej ustawiac to kolejnością odpowiedzialnoścową. W krótkich klasach to nie jest problem, ale jak by miała 40 linijek, to lepiej mieć wszystko odpowiedzialne za flexbox obok siebie.